screens/PaymentFormScreen.js

```diff
@@ -26,29 +32,53 @@ class PaymentFormScreen extends React.Component {
     getPercent: null
   }
 
+  componentDidMount() {
+    this.props.getfetchglobalOrderOffers()
+  }
+
   getPercent = percent => {
     this.setState({
       getPercent: percent
     })
   }
 
   render() {
-    const { order, navigation, handleSubmit, reset } = this.props
-    const discounts = [
-      {
-        label: 'Staff 20%',
-        value: 20
-      },
-      {
-        label: 'VIP 15%',
-        value: 15
-      },
-      {
-        label: 'Enter Discount',
-        value: ''
-      }
-    ]
+    const { order, navigation, handleSubmit, globalorderoffers } = this.props
+    //console.log(globalorderoffers)
+    // const discounts = [
+    //   {
+    //     label: 'Staff 20%',
+    //     value: 20
+    //   },
+    //   {
+    //     label: 'VIP 15%',
+    //     value: 15
+    //   },
+    //   {
+    //     label: 'Enter Discount',
+    //     value: ''
+    //   }
+    // ]
 
+    //discount arr by api
+    var discounts = []
+    var discountoffers = []
+    globalorderoffers !== undefined &&
+      globalorderoffers.map(globalorderoffer => {
+        discounts.push({
+          label: globalorderoffer.displayName,
+          //value: globalorderoffer.discountValue !== 0 ? globalorderoffer.discountValue : ''+ globalorderoffer.discountValue
+          value: globalorderoffer.discountValue
```

Comment:
i wonder if a checkbox value can be set like this: 
value: {
orderDiscount: globalorderoffer.offerName,
discount: globalorderoffer.discountValue
}